.pre-commit-config.yaml.jinja

```diff
@@ -13,10 +40,23 @@ exclude: |
 default_language_version:
   python: python3
 repos:
+  - repo: local
+    hooks:
+      # These files are most likely copier diff rejection junks; if found,
+      # review them manually, fix the problem (if needed) and remove them
+      - id: forbidden-files
+        name: forbidden files
+        entry: found forbidden files; remove them
+        language: fail
+        files: "\\.rej$"
   - repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: {{ repo_rev.black }}
     hooks:
       - id: black
+
+  {#- Old prettier required a separate step for the XML plugin because it
+      didn't support including plugins #}
+  {%- if odoo_version < 14 %}
   - repo: https://github.com/prettier/prettier
     rev: "1.19.1"
```

Comment:
@Yajo Why don't you use parameters `repo_rev.prettier` and `repo_rev.prettier_xml` also for version < 14? It's nice to see at the same place the versions of all the plugins used.  (Same applies to pylint and pylint-odoo)

Reply:
Yes, no problem.

Reply:
Finally I have not implemented this because it doesn't make sense with the other suggestion that I followed.